deploy.sh

```diff
@@ -86,6 +163,9 @@ fi
 HELM_COMMAND="${HELM_COMMAND} ${DEPLOY_NAME}"
 
 if [ "${HELM_ACTION}" == "install" ]; then
+    if [ "${OCI_REGISTRY}" == "true" ]; then
+        DEPLOY_CHART_PATH="${HELM_REPOSITORY}/${DEPLOY_CHART_PATH}/${DEPLOY_NAME}"
```

Comment:
Hi @LeoDiazL,
I'm having a hard time understanding this.
Theoretically, if my chart is stored at `oci://registry.io/organization/repository:version`
what parameters am I supposed to pass to the action?
And why does the Helm release name affect the chart path?